standalone docker image fails due to typo in volume mount

Following instructions in standalone README to launch standalone docker image.

```
Daves-MacBook-Pro:openwhisk dgrove$ ./core/standalone/start.sh 
...
ok: whisk auth set. Run 'wsk property get --auth' to see the new value.
ok: whisk API host set to http://openwhisk:3233
server still not ready - retrying
server still not ready - retrying
....
```

Got the logs...
```
Daves-MacBook-Pro:openwhisk dgrove$ docker logs openwhisk
+ java -Dwhisk.standalone.host.name=openwhisk -Dwhisk.standalone.host.internal=172.17.0.3 -Dwhisk.standalone.host.external=localhost -jar openwhisk-standalone.jar --no-browser

        ____      ___                   _    _ _     _     _
       /\   \    / _ \ _ __   ___ _ __ | |  | | |__ (_)___| | __
  /\  /__\   \  | | | | '_ \ / _ \ '_ \| |  | | '_ \| / __| |/ /
 /  \____ \  /  | |_| | |_) |  __/ | | | |/\| | | | | \__ \   <
 \   \  /  \/    \___/| .__/ \___|_| |_|__/\__|_| |_|_|___/_|\_\
  \___\/ tm           |_|
    
Git Commit: ddb2662, Build Date: 2020-10-27T14:26:23+0000
================================================================================
Running pre flight checks ...

Local Host Name: openwhisk
Local Internal Name: 172.17.0.3

[  OK   ] 'docker' cli found. (Docker version 18.06.3-ce, build d7080c1)
[  OK   ] 'docker' version 18.6.3 is newer than minimum supported 18.3.0
[FAILURE] 'docker' not found to be running. Failed to run 'docker info'
[  OK   ] 'wsk' cli found. (2019-09-23T17:46:38.323+0000)
[  OK   ] 'wsk' configured for namespace [guest].
[  OK   ] 'wsk' configured to connect to http://openwhisk:3233.
[  OK   ] Server port [3233] is free

================================================================================
[2020-10-29T00:14:02.558Z] [INFO] Starting OpenWhisk standalone on port 3233
[2020-10-29T00:14:03.024Z] [INFO] Slf4jLogger started
[2020-10-29T00:14:03.325Z] [INFO] Using [/root/.openwhisk/standalone/server-3233] as data directory
Exception in thread "main" java.lang.reflect.InvocationTargetException
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
[2020-10-29T00:14:03.757Z] [ERROR] [#tid_sid_unknown] [StandaloneDockerClient] Failed to determine docker client version: class org.apache.openwhisk.core.containerpool.docker.ProcessUnsuccessfulException - info: command was unsuccessful, code: 1 (unsuccessful), stdout: 18.06.3-ce, stderr: Cannot connect to the Docker daemon at unix:///var/run/docker.sock. Is the docker daemon running?
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
	at org.springframework.boot.loader.MainMethodRunner.run(MainMethodRunner.java:47)
	at org.springframework.boot.loader.Launcher.launch(Launcher.java:86)
	at org.springframework.boot.loader.Launcher.launch(Launcher.java:50)
	at org.springframework.boot.loader.JarLauncher.main(JarLauncher.java:51)
Caused by: org.apache.openwhisk.core.containerpool.docker.ProcessUnsuccessfulException: info: command was unsuccessful, code: 1 (unsuccessful), stdout: 18.06.3-ce, stderr: Cannot connect to the Docker daemon at unix:///var/run/docker.sock. Is the docker daemon running?
	at org.apache.openwhisk.core.containerpool.docker.ProcessRunner.$anonfun$executeProcess$6(ProcessRunner.scala:65)
	at org.apache.openwhisk.core.containerpool.docker.ProcessRunner$$Lambda$671/00000000909E56D0.apply(Unknown Source)
	at scala.concurrent.Future.$anonfun$flatMap$1(Future.scala:307)
	at scala.concurrent.Future$$Lambda$672/0000000090E32F90.apply(Unknown Source)
	at scala.concurrent.impl.Promise.$anonfun$transformWith$1(Promise.scala:41)
	at scala.concurrent.impl.Promise$$Lambda$674/0000000090E335D0.apply(Unknown Source)
	at scala.concurrent.impl.CallbackRunnable.run(Promise.scala:64)
	at akka.dispatch.BatchingExecutor$AbstractBatch.processBatch(BatchingExecutor.scala:55)
	at akka.dispatch.BatchingExecutor$BlockableBatch.$anonfun$run$1(BatchingExecutor.scala:92)
	at akka.dispatch.BatchingExecutor$BlockableBatch$$Lambda$673/00000000FC0043B0.apply$mcV$sp(Unknown Source)
	at scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.java:23)
	at scala.concurrent.BlockContext$.withBlockContext(BlockContext.scala:85)
	at akka.dispatch.BatchingExecutor$BlockableBatch.run(BatchingExecutor.scala:92)
	at akka.dispatch.TaskInvocation.run(AbstractDispatcher.scala:41)
	at akka.dispatch.ForkJoinExecutorConfigurator$AkkaForkJoinTask.exec(ForkJoinExecutorConfigurator.scala:49)
	at akka.dispatch.forkjoin.ForkJoinTask.doExec(ForkJoinTask.java:260)
	at akka.dispatch.forkjoin.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1339)
	at akka.dispatch.forkjoin.ForkJoinPool.runWorker(ForkJoinPool.java:1979)
	at akka.dispatch.forkjoin.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:107)

--- a/core/standalone/start.sh
+++ b/core/standalone/start.sh
@@ -20,7 +20,7 @@ shift
 docker run --rm -d \
   -h openwhisk --name openwhisk \
   -p 3233:3233 -p 3232:3232 \
-  -v //var/run/docker.sock:/var/run/docker.sock \
+  -v /var/run/docker.sock:/var/run/docker.sock \
  "$IMAGE" "$@"
 docker exec openwhisk waitready
 case "$(uname)" in

